### ISSUE TITLE
ender 3 v2 extender

### ISSUE THREAD

**Issue by geedog66** [smith3d/Marlin#13]
I have your firmware downloaded on my ender 3 v2 printer and i recently purchased the ender extender kit which makes my printer 400 x 400 x 500.. I still want to keep the features in your current firmware just be able to use it on my upgraded printer.. any help will be appreciated thanks..

Have you read Marlin's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/MarlinFirmware/Marlin/blob/master/.github/code_of_conduct.md

Do you want to ask a question? Are you looking for support? Please don't post here. Instead please use one of the support links at https://github.com/MarlinFirmware/Marlin/issues/new/choose

Before filing an issue be sure to test the "bugfix" branches to see whether the issue has been resolved.

-->

### Feature Workflow

1. [First Action]
2. [Second Action]
3. [and so on...]

#### Additional Information

* Provide pictures or links that demonstrate a similar feature or concept.
* See [How Can I Contribute](#how-can-i-contribute) for additional guidelines.

**Comment by jpbarcellos** [comment]
![image](https://user-images.githubusercontent.com/78943761/107725279-a2768c00-6cc4-11eb-9040-ba7438b2a9a8.jpeg)

**Comment by dcwalmsley** [comment]
@geedog66 

Do you have a BLTouch installed?  I am building off Smith3D's master a 400x400x250 version.  I can compile one for you with 400x400x500 if that will work with you.

Also need to know what version of Ender you are using.

**Comment by jpbarcellos** [comment]
yes, i do have bl touch.  I already did the compilation and everything is working normal.  the only thing I couldn't get it to work on was the filament sensor.  Can you help me with this? My Ender is v2 thanks

**Comment by Natech603** [comment]
I cant get my compilations to work using smith3d I have an ender 3 v2 with the 400x400x250 extender kit with bltouch v3. Ive tried a bunch of different firmware mods and different codes but cant seem to actually probe the proper coordinates. it works 100% fine on my stock build area e3v2 just not with the extender kit. If yall could help me out thaTD be more lit than christmas.
